src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlRedirection/XamlXmlReader.cs

```diff
@@ -21,81 +19,39 @@ public XamlXmlReader(XmlReader document, XamlSchemaContext context, XamlXmlReade
-					return XamlNodeType.EndMember;
-				case __ms::System.Xaml.XamlNodeType.Value:
-					return XamlNodeType.Value;
-				case __ms::System.Xaml.XamlNodeType.NamespaceDeclaration:
-					return XamlNodeType.NamespaceDeclaration;
-			}
-		}
+		public XamlNodeType NodeType => Convert(_unoReader.NodeType);
 
 		private XamlNodeType Convert(__uno::Uno.Xaml.XamlNodeType source)
-		{
-			switch (source)
+			=> source switch
 			{
-				default:
-				case __uno::Uno.Xaml.XamlNodeType.None:
-					return XamlNodeType.None;
-				case __uno::Uno.Xaml.XamlNodeType.StartObject:
-					return XamlNodeType.StartObject;
-				case __uno::Uno.Xaml.XamlNodeType.GetObject:
-					return XamlNodeType.GetObject;
-				case __uno::Uno.Xaml.XamlNodeType.EndObject:
-					return XamlNodeType.EndObject;
-				case __uno::Uno.Xaml.XamlNodeType.StartMember:
-					return XamlNodeType.StartMember;
-				case __uno::Uno.Xaml.XamlNodeType.EndMember:
-					return XamlNodeType.EndMember;
-				case __uno::Uno.Xaml.XamlNodeType.Value:
-					return XamlNodeType.Value;
-				case __uno::Uno.Xaml.XamlNodeType.NamespaceDeclaration:
-					return XamlNodeType.NamespaceDeclaration;
-			}
-		}
+				__uno::Uno.Xaml.XamlNodeType.StartObject => XamlNodeType.StartObject,
+				__uno::Uno.Xaml.XamlNodeType.GetObject => XamlNodeType.GetObject,
+				__uno::Uno.Xaml.XamlNodeType.EndObject => XamlNodeType.EndObject,
+				__uno::Uno.Xaml.XamlNodeType.StartMember => XamlNodeType.StartMember,
+				__uno::Uno.Xaml.XamlNodeType.EndMember => XamlNodeType.EndMember,
+				__uno::Uno.Xaml.XamlNodeType.Value => XamlNodeType.Value,
+				__uno::Uno.Xaml.XamlNodeType.NamespaceDeclaration => XamlNodeType.NamespaceDeclaration,
+				_ => XamlNodeType.None,
+			};
 
-		public object Value => XamlConfig.IsUnoXaml ? _unoReader.Value : _systemReader.Value;
+		public object Value => _unoReader.Value;
 
-		public XamlType Type => XamlConfig.IsUnoXaml ? XamlType.FromType(_unoReader.Type) : XamlType.FromType(_systemReader.Type);
+		public XamlType Type => XamlType.FromType(_unoReader.Type);
 
-		public int LineNumber => XamlConfig.IsUnoXaml ? _unoReader.LineNumber : _systemReader.LineNumber;
+		public int LineNumber => _unoReader.LineNumber;
 
-		public int LinePosition => XamlConfig.IsUnoXaml ? _unoReader.LinePosition : _systemReader.LinePosition;
+		public int LinePosition => _unoReader.LinePosition;
 
-		public XamlMember Member => XamlConfig.IsUnoXaml ? XamlMember.FromMember(_unoReader.Member) : XamlMember.FromMember(_systemReader.Member);
+		public XamlMember Member => XamlMember.FromMember(_unoReader.Member);
 
 		public NamespaceDeclaration Namespace 
-			=> XamlConfig.IsUnoXaml ? new NamespaceDeclaration(_unoReader.Namespace) : new NamespaceDeclaration(_systemReader.Namespace);
+			=> new NamespaceDeclaration(_unoReader.Namespace);
 
-		public void Dispose() => (XamlConfig.IsUnoXaml ? (IDisposable)_unoReader : _systemReader).Dispose();
+		public void Dispose() => ((IDisposable)_unoReader).Dispose();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Review this cast; in this project there's no type that extends 'Uno.Xaml.XamlXmlReader' and implements 'IDisposable'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5329947)